### PR TITLE
Merging in new image slides

### PR DESCRIPTION
**Pull request by BoopBoopBeepBoop**
Merging in new image slide types from upstream

### CODE REVIEW

File: lib/powerpoint.rb [hunk 1]
```diff
@@ -3,6 +3,8 @@
 require 'powerpoint/slide/intro'
 require 'powerpoint/slide/textual'
 require 'powerpoint/slide/pictorial'
+require 'powerpoint/slide/text_picture_split'
+require 'powerpoint/slide/picture_description'
```
Comment (zbelzer) [inline lib/powerpoint.rb:7]: At this point we might consider dynamic inclusion:

`Dir[File.expand_path("../powerpoint/*", __FILE__)].each { |f| require f }`

Comment (zbelzer) [inline lib/powerpoint.rb:7]: or `slide/*` but you get the gist

File: lib/powerpoint/slide/text_picture_split.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+      def file_type
+        File.extname(image_name).gsub('.', '')
+      end
+
+      def default_coords
+        start_x = pixle_to_pt(360)
+        default_width = pixle_to_pt(300)
+
+        return {} unless dimensions = FastImage.size(image_path)
```
Comment (zbelzer) [inline lib/powerpoint/slide/text_picture_split.rb:34]: I would do

``` ruby
if dimensions =
  ...
else
  {}
end
```

**Comment by BoopBoopBeepBoop** [comment]
Jenkins, test this please

**Pull request merged**